### DISCUSSION TITLE
Can I join to the team?

### DISCUSSION THREAD

**Discussion by orkunergun68** [ROM-builders/temporary#12117] [Q&A]
Cirrus is banned me on main account (same issue with Clhexftw) so switched to backup account ( Its not new actually) and I want to join the team .

Build with new acc : https://cirrus-ci.com/build/5744364432392192

Old acc @orkunergun

Telegram : t.me/orkunergun

Tree : https://gitlab.com/R9Lab

Kernel source : https://GitHub.com/orkunergun/LightningPlus

And I know git stuffs like cherry-pick commit rebase branch things etc. And I'm trying to keep my commit history clean.
Regards.

**Comment by Apon77** [comment]
> Cirrus is banned me on main account (same issue with Clhexftw) so switched to backup account ( Its not new actually) and I want to join the team .
> 
> Build with new acc : https://cirrus-ci.com/build/5744364432392192
> 
> Old acc @orkunergun
> 
> Telegram : t.me/orkunergun
> 
> Tree : https://gitlab.com/R9Lab
> 
> Kernel source : https://GitHub.com/orkunergun/LightningPlus
> 
> And I know git stuffs like cherry-pick commit rebase branch things etc. And I'm trying to keep my commit history clean. Regards.

since you were banned, there must be some reason

**Reply by orkunergun68** [reply]
> > Cirrus is banned me on main account (same issue with Clhexftw) so switched to backup account ( Its not new actually) and I want to join the team .
> > Build with new acc : https://cirrus-ci.com/build/5744364432392192
> > Old acc @orkunergun
> > Telegram : t.me/orkunergun
> > Tree : https://gitlab.com/R9Lab
> > Kernel source : https://GitHub.com/orkunergun/LightningPlus
> > And I know git stuffs like cherry-pick commit rebase branch things etc. And I'm trying to keep my commit history clean. Regards.
> 
> since you were banned, there must be some reason

but i have to join the team And I'm innocent If they hadn't banned me I would have tried with my main account also except using thier ci for building kernels and roms i did nothing else in cirrus
